src/pool/srv_iv.c

```diff
@@ -1173,6 +1173,8 @@ ds_pool_iv_prop_update(struct ds_pool *pool, daos_prop_t *prop)
 			    CRT_IV_SYNC_LAZY, true);
 	if (rc)
 		D_ERROR("pool_iv_update failed "DF_RC"\n", DP_RC(rc));
+
+	D_FREE(iv_entry);
```

Comment:
I don't think this is the right solution.   iv_entry is passed to pool_iv_update.  I think it might need to be freed on error path.   I mentioned this one to @wangdi1

Reply:
The same code in ds_pool_iv_prop_fetch() free it always. I think if it used inside pool_iv_update() it's coped into other location and here we should free it always.

Reply:
ds_pool_iv_prop_update() is asynchronou(SYNC_LAZY)s, so iv_entry needs to be kept after the call, until the callback is called. see sync_comp_cb.
And ds_pool_iv_prop_fetch() is synchronous, so iv_entry can be freed

Reply:
Ah, just recheck the code, iv_entry is actually cloned in iv_op if it is asynchronous. so it is safe to free iv_entry(). Thanks.